samples/wifi/promiscuous/README.rst

```diff
@@ -0,0 +1,307 @@
+      [00:00:07.712,585] <inf> wifi_connect: Connected
+      [00:00:07.839,050] <inf> net_dhcpv4: Received: 192.165.100.161
+      [00:00:07.839,202] <inf> net_config: IPv4 address: 192.165.100.161
+      [00:00:07.839,202] <inf> net_config: Lease time: 86400 seconds
+      [00:00:07.839,233] <inf> net_config: Subnet: 255.255.0.0
+      [00:00:07.839,294] <inf> net_config: Router: 192.165.100.68
+      [00:00:07.839,385] <inf> wifi_connect: DHCP IP address: 192.165.100.161
+      [00:00:07.842,468] <inf> wifi_connect: ==================
+      [00:00:07.842,498] <inf> wifi_connect: State: COMPLETED
+      [00:00:07.842,529] <inf> wifi_connect: Interface Mode: STATION
+      [00:00:07.842,529] <inf> wifi_connect: Link Mode: WIFI 4 (802.11n/HT)
+      [00:00:07.842,559] <inf> wifi_connect: SSID: AAAAAAAA
+      [00:00:07.842,590] <inf> wifi_connect: BSSID: XX:XX:XX:XX:XX:XX
+      [00:00:07.842,620] <inf> wifi_connect: Band: 2.4GHz
+      [00:00:07.842,620] <inf> wifi_connect: Channel: 11
+      [00:00:07.842,620] <inf> wifi_connect: Security: OPEN
+      [00:00:07.842,651] <inf> wifi_connect: MFP: Disable
+      [00:00:07.842,651] <inf> wifi_connect: RSSI: -57
+      [00:00:07.842,681] <inf> wifi_connect: TWT: Not supported
+      [00:00:07.844,024] <inf> promiscuous: Network capture of Wi-Fi traffic enabled
+      [00:00:07.844,177] <inf> promiscuous: Wi-Fi promiscuous mode RX thread started
+      [00:00:12.869,476] <inf> promiscuous: Management Frames:
+      [00:00:12.869,506] <inf> promiscuous:   Beacon Count: 48
+      [00:00:12.869,506] <inf> promiscuous:   Probe Request Count: 0
+      [00:00:12.869,506] <inf> promiscuous:   Probe Response Count: 190
+      [00:00:12.869,537] <inf> promiscuous:   Action Count: 0
+      [00:00:12.869,537] <inf> promiscuous: Control Frames:
+      [00:00:12.869,567] <inf> promiscuous:    ACK Count 0
+      [00:00:12.869,567] <inf> promiscuous:    RTS Count 0
+      [00:00:12.869,567] <inf> promiscuous:    CTS Count 0
+      [00:00:12.869,567] <inf> promiscuous:    Block Ack Count 0
+      [00:00:12.869,598] <inf> promiscuous:    Block Ack Req Count 0
+      [00:00:12.869,628] <inf> promiscuous:    CF End Count 0
+      [00:00:12.869,628] <inf> promiscuous: Data Frames:
+      [00:00:12.869,628] <inf> promiscuous:   Data Count: 9
+      [00:00:12.869,628] <inf> promiscuous:   QoS Data Count: 0
+      [00:00:12.869,659] <inf> promiscuous:   Null Count: 16
+      [00:00:12.869,659] <inf> promiscuous:   QoS Null Count: 8
+      [00:00:12.869,659] <inf> promiscuous: Unknown Frames:
+      [00:00:12.869,659] <inf> promiscuous:   Unknown Management Frame Count: 1
+      [00:00:12.869,659] <inf> promiscuous:   Unknown Control Frame Count: 0
+      [00:00:12.869,689] <inf> promiscuous:   Unknown Data Frame Count: 0
+      [00:00:12.869,689] <inf> promiscuous:   Reserved Frame Count: 0
+      [00:00:12.869,689] <inf> promiscuous:   Unknown Frame Count: 0
+
+Offline net capture
```

Comment:
The **Offline net capture** section is in the monitor sample as well, so I suggest creating a include text file that can be used across the samples.